rskj-core/src/main/java/co/rsk/peg/lockingcap/LockingCapSupportImpl.java

```diff
@@ -45,7 +46,11 @@ public Optional<Coin> getLockingCap() {
     }
 
     @Override
-    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) {
+    public boolean increaseLockingCap(Transaction tx, Coin newLockingCap) throws BridgeIllegalArgumentException {
```

Comment:
I wonder if it's the best idea to throw an `BridgeIllegalArgumentException` here. Since that exception is on an upper level package we are creating a circular dependency.

Maybe we should throw a specific locking cap related exception and let the Bridge capture it and throw the BridgeIllegalArgumentException then?

Reply:
agree

Reply:
@marcos-iov It's not a circular dependency. It's being used in only one direction. We are using an `Exception` at one upper level where the LockingCapSupport is, but in an exception should be good.
<img width="539" alt="image" src="https://github.com/user-attachments/assets/984538ed-00f7-4a87-b7b6-3b56fed7315a">

Reply:
@marcos-iov Done!